apps/web/app/(booking-page-wrapper)/layout.tsx

```diff
@@ -0,0 +1,25 @@
+import { cookies, headers } from "next/headers";
+
+import { buildLegacyCtx } from "@lib/buildLegacyCtx";
+
+import PageWrapper from "@components/PageWrapperAppDir";
+
+import { ssrInit } from "@server/lib/ssr";
+
+export default async function BookingPageWrapperLayout({ children }: { children: React.ReactNode }) {
+  const h = headers();
+  const nonce = h.get("x-nonce") ?? undefined;
+  const context = buildLegacyCtx(headers(), cookies(), {}, {});
+  const ssr = await ssrInit(context);
+
+  return (
+    <PageWrapper
+      isBookingPage={true}
```

Comment:
the reason I created a separate route group called `(booking-page-wrapper)`

Reply:
Oh I love how simple it is with `isBookingPage={true}` here